frontend/packages/kubevirt-plugin/integration-tests-cypress/utils/const/errorString.ts

```diff
@@ -1,2 +1,3 @@
 export const bootsourceError =
   'This disk does not have a source defined and can not be used as a boot source';
+export const errorOccurred = 'An error occurred';
```

Comment:
nitpick: These constants should use the constants naming convention (`ERROR_OCCURRED`).

Reply:
make sense, updated.